hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java

```diff
@@ -322,7 +330,8 @@ public void checkPropertiesAccess() {
     public void checkPropertyAccess(String key) {
         if (!callFromAcceptClassLoaders() && callFromGremlin() &&
             !WHITE_SYSTEM_PROPERTYS.contains(key) && !callFromBackendHbase() &&
-            !callFromRaft() && !callFromSofaRpc()) {
+            !callFromBackendRocksDB() && !callFromRaft() &&
```

Comment:
unnecessary?

Reply:
maybe exist some dup part, but indeed need it.